tests/io/test_dataset.py

```diff
@@ -0,0 +1,69 @@
+from pandas import read_hdf
+from sleap_io import (
+    from_pointsarray,
+    from_instance,
+    read_hdf5,
+    read_videos,
+    read_skeleton,
+    read_tracks,
+    read_instances,
+    read_metadata,
+    read_points,
+    read_pred_points,
+    Video,
+    Skeleton,
+    Edge,
+    Node,
+    Instance,
+    LabeledFrame,
+    Track,
+    Point,
+    PredictedPoint,
+    PredictedInstance,
+)
+import numpy as np
+from typing import List
+
+# L136, L178, L255
+
+skeleton = Skeleton(
+    nodes=[Node("head"), Node("thorax"), Node("abdomen")],
+    edges=[
+        Edge(source=Node("head"), destination=Node("thorax")),
+        Edge(source=Node("thorax"), destination=Node("abdomen")),
+    ],
+)
+instance1 = Instance(skeleton=skeleton, points={"head": Point(x=0, y=0)})
+numpy_array1 = np.array([[1, 1], [2, 2], [3, 3]], dtype="float32")
+numpy_array2 = np.array(
+    [[1, 1, True, False], [2, 2, True, False], [3, 3, True, False]], dtype="float32"
+)
+instance2 = from_pointsarray(points=numpy_array1, skeleton=skeleton)
+instance3 = from_pointsarray(points=numpy_array2, skeleton=skeleton)
```

Comment:
Move this inside your function. If you need fixtures (you believe these will be reused), then add them as fixtures.